content/documentation/getting-started/index.adoc

```diff
@@ -269,25 +269,27 @@ The above is actually the default when Kiali is installed on Kubernetes - so to
 
 === Login Options
 
-Kiali supports three different login options.
+Kiali supports four different login options.
```

Comment:
perhaps "several" will be better since it will not require to maintain this line anymore :)

Reply:
done.